docs/resources/access_application.md

```diff
@@ -52,6 +52,7 @@ resource "cloudflare_access_application" "staging_app" {
 
 ### Required
 
+- `type` (String) The application type. Available values: `self_hosted`, `saas`, `ssh`, `vnc`, `bookmark`. Defaults to `self_hosted`.
```

Comment:
these changes will first need to be made to the respective file in `templates` before running `make docs` that will output this. further details can be found at https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md